packages/repo/src/verify.ts

```diff
@@ -153,4 +155,87 @@ export const verifyCommitPath = async (
   return updates
 }
 
+export const verifyProofs = async (
```

Comment:
I didn't add the equivalent sync methods for these because proofs like these some more one off & im not sure consumers will want to save them. Just seemed a bit superfluous.

On the otherhand if they don't, they could always just pass in a memory blockstore 🤔